en/editor/console/index.md

```diff
@@ -0,0 +1,28 @@
+# Console
+
+![main](index/main.jpg)
+
+**Console** display errors, warnings, or other log information generated by the editor and engine. Information of different important levels is displayed in different colors.
+
+## Abourt log
+The log system is based on `js` **console** model. When the editor is running, will captures the necessary logs printed out of the **console** in each process and distinguishes them by color.
```

Comment:
1. The log system is based on `js` 
--> The log system is based on the `js`

2. When the editor is running, will captures the necessary logs printed out of the **console**
--> When the editor is running, logs will be captured and printed out to the **console**